internal/eval/eval_test.go

```diff
@@ -90,7 +90,7 @@ func TestEval(t *testing.T) {
 	testEvalHLEN(t, store)
 	testEvalSELECT(t, store)
 	testEvalLLEN(t, store)
-	testEvalGETEX(t, store)
```

Comment:
This shouldn't be removed, right?

Reply:
It's called earlier on line 52

Reply:
https://github.com/DiceDB/dice/blob/b96c572f580249741e2712fe393e5ccb7155557e/internal/eval/eval_test.go#L52C1-L52C25